git_machete/git_operations.py

```diff
@@ -89,9 +89,7 @@ def get_default_editor(self) -> Optional[str]:
                     if name != "$" + git_machete_editor_var and self.get_config_or_none('advice.macheteEditorSelection') != 'false':
                         sample_alternative = 'nano' if editor.startswith('vi') else 'vi'
                         sys.stderr.write(
-                            fmt(f"Opening <b>{editor_repr}</b>.\n",
```

Comment:
Whoops pls re-split this line

Reply:
(comment left)

Reply:
When I am splitting this, it does not pass pep8 check.

Reply:
🤔 what's the specific error? pls note that for the sake of sanity, in https://github.com/VirtusLab/git-machete/blob/master/tox.ini#L29 we've got the `E501` check (_lines must be <=80 chars) disables

Reply:
(otherwise PR ready to approve AFAICS)

Reply:
Seems that pep8 passed now, okay...

Reply:
Yes :)